pkg/interface/src/views/landscape/components/JoinGroup.tsx

```diff
@@ -75,12 +77,12 @@ export function JoinGroup(props: JoinGroupProps): ReactElement {
     await api.groups.join(ship, name);
     try {
       await waiter((p: JoinGroupProps) => {
```

Comment:
You're going to want to pass the hook values into `useWaitForProps` and then check there